docs/architecture/anti-entropy/hinted-handoff.rst

```diff
@@ -98,6 +98,7 @@ Examples:
     $ curl -X GET "http://127.0.0.1:10000/hinted_handoff/sync_point?timeout=-1"
     {"message": "Missing mandatory parameter 'id'", "code": 400}
 
+See also the API reference `here </reference/api-reference/#hinted-handoff>`_.
```

Comment:
I get 404 for this link (in preview)

Reply:
Hm, that's right... It seems the issue boils down to using a "regular" link instead of `:ref:` or something similar. I was looking for a way to point to a specific section, but none of the solutions worked except for this one. If `#hinted-handoff` is not critical, I'd suggest dropping it and changing the link to `:ref:`here </reference/api-reference/>`. But if there's a better way, I'm open for suggestions

Reply:
@dgarcia360 please advise

Reply:
This method of defining references to anchors may cause issues with multiversion support, as it requires adding the current version (latest or branch name) to the link.

The solution is to create a custom role to support cross-references to anchors in pages that generated with Markdown or reStructuredText, like the API references. I’ve added this to my to-do list.

While we don't have this custom role, there are two workarounds:

**1. Option A:** Use the complete URL

```
`here <https://opensource.docs.scylladb.com/stable/reference/api-reference#hinted-handoff>`_
```

*Issue:* Users will always be redirected to the latest stable version, and local previews will redirect to the production page.

**2. Option B:** Link to the complete API reference

```
:doc:`here </reference/api-reference>`
```

*Issue:* Users will need to manually locate the "hinted handoff" section within the API reference.

Reply:
Ouch. Both are not great.

Reply:
> The solution is to create a custom role to support cross-references to anchors in pages that NOT generated with Markdown or reStructuredText, like the API references. I’ve added this to my to-do list.

@dgarcia360 Is there a GitHub issue for this? I'd like to be able to track this somehow so that we know when we can come back to this PR.